ctrlf.el

```diff
@@ -1147,6 +1146,12 @@ See `ctrlf-mode-bindings' to customize."
         (advice-remove #'minibuffer-message
                        #'ctrlf--minibuffer-message-condense)))))
 
+;;;###autoload
+(define-globalized-minor-mode ctrlf-global-mode ctrlf-mode ctrlf-mode-turn-on)
```

Comment:
This is not needed, calling a mode function with no arguments enables it unconditionally in modern Emacs versions.

```suggestion
(define-globalized-minor-mode ctrlf-global-mode ctrlf-mode)
```

Reply:
Fixed. We still need to provide a function for turning on, so it is:
``
(define-globalized-minor-mode ctrlf-mode ctrlf-local-mode ctrlf-local-mode)
```